build.go

```diff
@@ -313,6 +316,63 @@ func generateSquashfsLayer(oci casext.Engine, name string, author string, opts *
 	return nil
 }
 
+// SaveLayer stores the final layers into a separate location based on the content of
+// the stackerfile, this is useful to avoid an extra manual step to upload build results
+// and also in case of caching in between stacker builds
+// The logic should work for both Docker registry destination and OCI layout destinations
+// In case of OCI layout destinations the tag will be included in the layer name
+func SaveLayer(opts *BuildArgs, sf *Stackerfile, name string) error {
+	if len(sf.buildConfig.SaveUrl) == 0 {
+		fmt.Printf("layer %s will not be save since it doesn't have a save URL\n", name)
```

Comment:
Can you drop this? It sounds scary, and it seems like mostly normal operation.

Reply:
This function is not be called in normal operations, only if you use the command line option --remote-save.
Should I remove it?

Reply:
Oh, right. I think this is really just an implementation of the check I wanted below; can we do it earlier though so we can fail faster (or just refactor it out per discussion below)?

Reply:
Done